openmdao/docs/usr-guide/tutorials/recording.rst

```diff
@@ -543,3 +542,147 @@ This code prints out the following:
+using the option:
+
+::
+
+    recorder.options['record_derivs'] = True
+
+then the derivatives are also recorded to the case recording file.
+
+.. testsetup:: reading_derivs
+
+    import os
+    if os.path.exists('paraboloid'):
+        os.remove('paraboloid')
+
+    from openmdao.api import IndepVarComp, Component, Group, Problem, ScipyOptimizer, SqliteRecorder
+
+    class Paraboloid(Component):
+        """ Evaluates the equation f(x,y) = (x-3)^2 + xy + (y+4)^2 - 3 """
+
+        def __init__(self):
+            super(Paraboloid, self).__init__()
+
+            self.add_param('x', val=0.0)
+            self.add_param('y', val=0.0)
+
+            self.add_output('f_xy', val=0.0)
+
+        def solve_nonlinear(self, params, unknowns, resids):
+            """f(x,y) = (x-3)^2 + xy + (y+4)^2 - 3
+            Optimal solution (minimum): x = 6.6667; y = -7.3333
+            """
+
+            x = params['x']
+            y = params['y']
+
+            unknowns['f_xy'] = (x - 3.0) ** 2 + x * y + (y + 4.0) ** 2 - 3.0
+
+        def linearize(self, params, unknowns, resids):
+            """ Jacobian for our paraboloid."""
+
+            x = params['x']
+            y = params['y']
+            J = {}
+
+            J['f_xy', 'x'] = 2.0 * x - 6.0 + y
+            J['f_xy', 'y'] = 2.0 * y + 8.0 + x
+            return J
+
+
+    # to keep the output of the run from doctest which does not handle output from setup well!
+    import os
+    import sys
+    f = open(os.devnull, 'w')
+    sys.stdout = f
+
+    top = Problem()
+
+    root = top.root = Group()
+
+    root.add('p1', IndepVarComp('x', 3.0))
+    root.add('p2', IndepVarComp('y', -4.0))
+    root.add('p', Paraboloid())
+
+    root.connect('p1.x', 'p.x')
+    root.connect('p2.y', 'p.y')
+
+    top.driver = ScipyOptimizer()
+    top.driver.options['optimizer'] = 'SLSQP'
+
+    top.driver.add_desvar('p1.x', lower=-50, upper=50)
+    top.driver.add_desvar('p2.y', lower=-50, upper=50)
+    top.driver.add_objective('p.f_xy')
+
+    recorder = SqliteRecorder('paraboloid')
+    recorder.options['record_params'] = True
+    recorder.options['record_metadata'] = True
+    recorder.options['record_derivs'] = True
+    top.driver.add_recorder(recorder)
+
+    top.setup()
+    top.run()
+
+    top.cleanup()
+
+.. testoutput:: reading_derivs
+   :hide:
+   :options: -ELLIPSIS, +NORMALIZE_WHITESPACE
+
+    Optimization terminated successfully.    (Exit mode 0)
+                Current function value: [-27.33333333]
+                Iterations: 5
+                Function evaluations: 6
+                Gradient evaluations: 5
+    Optimization Complete
+    -----------------------------------
+
+
+    Minimum of -27.333333 found at (6.666667, -7.333333)
+
+
+
+.. testcode:: reading_derivs
+
+    import sqlitedict
+    from pprint import pprint
+
+    db = sqlitedict.SqliteDict( 'paraboloid', 'openmdao_derivs' )
+
+
+The name of the sqlite table containing the derivatives is called `openmdao_derivs`.
```

Comment:
I probably missed discussion around this..   but why do you need a whole other table... can't you just have `Derivatives` in the normal table? (it could just be empty if the derivs weren't recorded?)